modules/options/src/main/scala/scala/build/options/JmhOptions.scala

```diff
@@ -1,9 +1,28 @@
 package scala.build.options
 
+import scala.build.internal.Constants
+
+/** @param jmhVersion
+  *   the version of JMH to be used in the build
+  * @param enableJmh
+  *   toggle for enabling JMH dependency handling in the build (overrides [[runJmh]] when disabled)
+  * @param runJmh
+  *   toggle for whether JMH should actually be runnable from this build
```

Comment:
Could you explain a bit more in the comment what this flag does?

Reply:
done